site/pages/api/delete-user.api.ts

```diff
@@ -0,0 +1,20 @@
+import { NextApiResponse } from "next";
+import { deleteAdminUser } from "../../data/cognito";
+import { DeleteUserApiRequest } from "../../interfaces";
+import { redirectTo, redirectToError } from "../../utils/apiUtils";
+
+const deleteUser = async (req: DeleteUserApiRequest, res: NextApiResponse): Promise<void> => {
```

Comment:
We need to restrict access to this api endpoint so only authenticated admins can call it, if we update line 173 in `middleware.api.ts` to:

```ts
if (request.nextUrl.pathname.startsWith("/admin/") || request.nextUrl.pathname.startsWith("/api/admin/")) {
```

and move this file into an admin folder then that should do the job, the action endpoint for the form will need updating too

Reply:
done